README.md

```diff
@@ -252,6 +254,25 @@ node {
 }
 ```
 
+It is also possible to use it as a UsernamePassword credentials, to do so, tag the secret with the desired `username`:  
+```bash
+az keyvault secret set --vault-name my-vault --name github-pat --value my-pat --tags username=github-user
+```
+
+Scripted Pipeline:  
```

Comment:
@timja Would it make sense to include an example for Scripted Pipeline here as well?

Reply:
probably not needed, there's nothing special about it